packages/labs/virtualizer/package.json

```diff
@@ -35,6 +35,12 @@
     "./polyfills/resize-observer-polyfill/ResizeObserver.js": {
       "default": "./polyfillLoaders/ResizeObserver.js"
     },
+    "./support/method-interception.js": {
+      "default": "./support/method-interception.js"
+    },
```

Comment:
this might be more of a tangent since it looks like it's been like this for a while but is it fine that we don't have `"types"` pointing to the `d.ts` files here like our other packages?

Reply:
Went ahead and added them.

Reply:
I tried testing it and it does seem look like TS is able to find it if the path and file location matches, but I think it's nice to be explicit too.